src/components/AutoComplete/AutoComplete.js

```diff
@@ -0,0 +1,39 @@
+import React, { memo, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import checkProps from '@jam3/react-check-extra-props';
+
+import typingAhead from 'typing-ahead';
+
+const AutoComplete = ({
+  data,
+  className,
+  autoCompleteResult
+}) => {
+  const [model, setModel] = useState(null);
+
+  function onInputChange(event) {
```

Comment:
You could destructure this partially here
```suggestion
  function onInputChange({target}) {
```

Reply:
Good call ! I have de-structure the event object two-level deep to get the value variable directly